### PR TITLE
System settings module

### PR DESCRIPTION
**Pull request by ChristianPerez34**
Set up general architecture of system settings. The system will be able to run in the following configurations: Development, Testing, and Production.

Development configuration is, as the name suggests, for development purposes

Testing configuration is planned to be used in conjunction with GitHub to run a CI/CD service

Production configuration should be used when the app is deployed into production.

If required settings are not set, the app will display an error message in the terminal so that the user may set the environment variables.

[Wiki](https://github.com/T4Future/T4Loans-API/wiki/Usage)

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1,5 @@
 flask==1.1.1
-flask-restful==0.3.8
\ No newline at end of file
+flask-restful==0.3.8
+flask-sqlalchemy==2.4.1
+pymysql==0.9.3
+black
```
Comment (alejoreyes96) [inline requirements.txt]: black is a formatter used for our preference of development. Should  it be included in the req file, given it is not an actual requirement to run the project?

Comment (ChristianPerez34) [reply requirements.txt]: Cierto es

File: settings/config.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import os
+
+
+class Config:
+    DEBUG = False
+    TESTING = False
+    SECRET_KEY = os.getenv("SECRET_KEY", "t4loans-wow")
+    JWT_SECRET_KEY = os.getenv("JWT_SECRET_KEY", "t4loans-wow")
+    SQLALCHEMY_DATABASE_URI = os.getenv("DATABASE_URI", None)
+
```
Comment (alejoreyes96) [inline settings/config.py]: add: `SQLALCHEMY_TRACK_MODIFICATIONS = False`
To suppress a warning given when running;

Warning: `FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead`

Comment (alejoreyes96) [inline settings/config.py]: add: `SQLALCHEMY_TRACK_MODIFICATIONS = False`
To suppress a warning given when running;

Warning: `FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead`

**Pull request merged**